src/main/java/digital/slovensko/autogram/core/eforms/EFormUtils.java

```diff
@@ -220,23 +222,30 @@ public static String getNamespaceFromEformXml(Document xml) {
         return null;
     }
 
-    public static byte[] getResource(String url) {
+    public static byte[] getResource(String url) throws ServiceUnavailableException {
         var offlineFileLoader = new FileCacheDataLoader();
         offlineFileLoader.setCacheExpirationTime(21600000);  // 6 hours
         offlineFileLoader.setDataLoader(new CommonsDataLoader());
 
         DSSDocument xsltDoc;
         try {
             xsltDoc = offlineFileLoader.getDocument(url);
+        } catch (DSSExternalResourceException e) {
+            String statusCode = e.getCause().getMessage().replaceAll("[^0-9]", "");
```

Comment:
Toto je ale riadny magic. Vyhodíme z nejakej error hlášky všetky nie-čísla a ak nám zostala na začiatku "5", považujeme to za 5xx response? Toto treba lepšie vymyslieť.

Reply:
Rozmýšľal som nad tým a musíme zo správy `Not acceptable HTTP Status (HTTP status code : 503 / reason : Service Unavailable` získať status kód. Myslíš, že regex, ktorý bude matchovat `HTTP status code : XXX` bude lepšia voľba?

Reply:
určite áno. Najlepšie to spraviť pomocou capture group v regexe - že to matchne aj s textom okolo, ale vieš potom vybrať hodnotu presne tej groupy, čo bude iba ten samotný kód.